docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,103 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
+Machine learning systems used in the real world are inherently sociotechnical
+systems, which include both technologies and social actors. Designers of machine
+learning systems who fail to consider the way in which social contexts and technologies
+are interrelated are at risk of falling into "abstraction traps". In this section,
+we explain what those traps are, and give some suggestions on how you can avoid them.
+
+In "Fairness and Abstraction in Sociotechnical Systems" Selbst et al. [#4]_
+identify failure modes that arise from applying abstraction. They identify
+them as:
+
+* *The Solutionism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Formalism Trap*
+
+* *The Portability Trap*
+
+* *The Framing Trap*
+
+These traps are inspired by the observation that each of these traps are the
+result of failing to consider the way in with social context and technology
+are interrelated, as well as a deeper understanding of "the social" in order to
+solve problems Ackerman [#5]_.
+
+The Solutionism Trap
+^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when data scientists assume that the best solution to a problem involves
+technology, and fail to recognize other possible solutions outside of this realm.
+One area where this manifests in is within contexts in which the definition "fairness"
```

Comment:
I am not sure if I understand what you mean with this sentence. Isn't the definition of "fairness" always dynamic / dependent on a political context? The sentence is also a bit hard to parse.

Reply:
Reworded as

`This trap occurs when we assume that the best solution to a problem
may involve technology, and fail to recognize other possible solutions outside of
this realm. This trap also occurs as definitions of fairness are
contested and changed and models become obsolete.`

Reply:
Reworded, and incorporated "fairness" as dynamic and dependent on a political context.